### DISCUSSION TITLE
Cara menambahkan nama pasangan (suami istri) di pohon keluarga

### DISCUSSION THREAD

**Discussion by nimdasx** [nafiesl/silsilah#115] [Q&A] [answered]
sebelumnya, terima kasih saya ucapkan ke mas nafiesl dan tim, aplikasi ini sangat berguna sekali.   
mohon ijin bagaimana cara modifikasi di codingnya jika saya ingin menambahkan nama pasangan (suami/istri) di pohon keluarga.   
kebetulan saya belum pernah menggunakan laravel.   

terima kasih

**Comment by nafiesl** [comment, marked as answer]
Halo mas @nimdasx. Alhamdulillah semoga bermanfaat mas.

Terkait pertanyaannya, saya sendiri saat ini belum ketemu metode yang cocok untuk diterapkan di project ini (tepatnya belum sempat ngulik lagi). Ada 1 project opensoruce serupa, yang secara logic dia melihat dari project ini. 

https://github.com/MGeurts/genealogy

Di project ini, developernya mengubah bentuk tampilan family tree menjadi berpasangan suami isteri (lihat screenshot dan demonya). Nah mas bisa coba pelajari source codenya.

**Reply by nimdasx** [reply]
terima kasih mas
